src/python/WMCore/Services/AlertManager/AlertManagerAPI.py

```diff
@@ -0,0 +1,94 @@
+"""
+
+AlertManagerAPI - send alerts to MONIT AlertManager via API calls
+"""
+
+from __future__ import division
+from datetime import timedelta, datetime
+import socket
+import json
+import logging
+
+from WMCore.Services.pycurl_manager import RequestHandler
+from Utils.Timers import LocalTimezone
+
+
+class AlertManagerAPI(object):
+    """
+    A class used to send alerts via the MONIT AlertManager API
+    """
+
+    def __init__(self, alertManagerUrl):
+        self.alertManagerUrl = alertManagerUrl
+        # sender's hostname is added as an annotation
+        self.hostname = socket.gethostname()
+        self.mgr = RequestHandler()
+        self.ltz = LocalTimezone()
+
+    def send(self, alertName, severity, summary, description, tag="wmcore", service="", endSecs=600, generatorURL=""):
```

Comment:
Perhaps, we could give an even more explicit name to this method, e.g. `sendAlert`. What do you think?
The docstring is also missing the tag parameter. 
Last comment here is actually a question, as I look at it, I wonder if we should simply make `service` a mandatory argument?

Reply:
Service being mandatory is probably a good idea. We could go one step further and make a list of "valid" services to check against, since the service name will be used to route alerts.

Reply:
I like the idea of validating the service against a list of known and expected services (properly capitalized or not). The only problem will be if others, e.g. CRAB, decide to use this same module. 
Ideally the list of valid service names should be configurable, but I fail to see how to enforce that ... and the only feasible logic I see is hard-wiring it in the source code and updating it in the future, if needed. What are your thoughts here?

Reply:
For now we should probably just trust ourselves to use the correct names instead of putting in additional checks. Something to think about if we find we are unable to handle that.